packages/manager/apps/catalog-revamp/src/utils/test-products.mock.json

```diff
@@ -0,0 +1,56 @@
+{
```

Comment:
Don't mix utils and mocks : mocks are not part of the production code. Put them in a `mock` folder or in the test folder.

Reply:
Done